index.js

```diff
@@ -1,12 +1,81 @@
 const express = require('express');
 const helmet = require('helmet');
 
+const knex = require('knex');
+
+const knexConfig = {
+  client: 'sqlite3',
+  connection: {
+    filename: './data/lambda.sqlite3'
+  },
+  useNullAsDefault: true
+}
+
+const db = knex(knexConfig);
+
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
 
 // endpoints here
+server.get('/api/zoos', (req, res) => {
+  db('zoos')
+    .then(animals => {
+      res.status(200).json(animals);
+    })
+    .catch(err => {
+      res.status(500).json(err);
+    });
+});
+
+server.get('/api/zoos/:id', (req, res) => {
+  db('zoos')
+    .where({ id: req.params.id })
+    .first()
+    .then(zoo => {
+      if(zoo) {
+        res.status(200).json(zoo);
+      } else {
+        res.status(404).json({ message: 'Zoo not found' });
+      }
+    })
+    .catch(err => {
+      res.status(500).json(err);
+    });
+});
+
```

Comment:
you probably will want to return an error before you even execute the promise, for the no name param, you already know, the name is going to be required as part of your schema, that way, right when you read the req.body, if it doesnt have a name, reject, dont even let it go further.